crt_portal/cts_forms/model_variables.py

```diff
@@ -1,58 +1,59 @@
 """Setting the variables that can be reused in models and forms for readability and reuse"""
 
-from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy as _
 
 PRIMARY_COMPLAINT_CHOICES = (
     ('workplace', _('Workplace discrimination or other employment-related problem')),
```

Comment:
Confirmed that these Primary Issue responses are presented correctly to the front-end and stored correctly in the database: 

### Frontend
<img width="653" alt="laboral-frontend" src="https://user-images.githubusercontent.com/3209501/69381869-55e80080-0c7b-11ea-962b-56e64244ffe6.png">

### Database
<img width="1437" alt="laboral-workplace" src="https://user-images.githubusercontent.com/3209501/69381868-55e80080-0c7b-11ea-83ea-ed2f11db0476.png">